View/Screens/PlaylistMenuScreen.xaml.cs

```diff
@@ -38,12 +40,15 @@ private void PlaylistsRow_Click(object sender, MouseButtonEventArgs e)
         {
             var dataGridRow = (DataGridRow)sender;
             var selectedPlaylist = (Playlist)dataGridRow.Item;
+            
 
             MainWindow.MenuItemRoutedEvent?.Invoke(this, new MenuItemRoutedEventArgs
             {
                 ScreenName = ScreenNames.PlaylistScreen,
                 Playlist = selectedPlaylist
             });
+            SongListDataContext.Instance.ScreenName = ScreenNames.PlaylistScreen;

```

Comment:
Ditt doe je toch ook al in` MainWindow.SetScreen()`?

Reply:
Jep goeie, over het hoofd gezien